doc/reference/storage_pure.md

```diff
@@ -0,0 +1,100 @@
+(storage-pure)=
+# PureStorage - `pure`
+
+[PureStorage](https://www.purestorage.com/) is a software-defined storage solution. It offers the consumption of redundant block storage across the network.
+
+LXD offers access to PureStorage storage clusters using either iSCSI or NVMe/TCP.
+In addition, PureStorage offers copy-on-write snapshots, thin provisioning, and other features.
+
+To use PureStorage, ensure the required kernel modules for the selected protocol are installed on your host system.
+For iSCSI, the iSCSI CLI named `iscsiadm` needs to be installed in addition to the required kernel modules.
+
+## Terminology
+
+Each storage pool created in LXD using a PureStorage driver represents a PureStorage *pod*, which is an abstraction that groups multiple volumes under a specific name.
+Another benefit of using PureStorage pods is that they can be linked with multiple PureStorage arrays to provide additional redundancy.
+LXD creates volumes within a pod that is identified by the storage pool name.
+When the first volume needs to be mapped to a specific LXD host, a corresponding PureStorage host is created with the name of the LXD host and a suffix of used protocol.
```

Comment:
```suggestion
When the first volume needs to be mapped to a specific LXD host, a corresponding PureStorage host is created with the name of the LXD host and a suffix of the used protocol.
```

Reply:
Can we add an example here of what that PureStorage host name might look like? Like, "For example, a PureStorage host name might be ...".